pages/embedded-portal.tsx

```diff
@@ -26,20 +26,33 @@ import StepList, { Step } from "../components/shared/step-list";
 import Workspace from "../components/embedded-portal/workspace";
 import { theme } from "../lib/theme";
 import { useFlashMessages, useOnClickOutside } from "../lib/hooks/usehooks";
+import { Flatfile } from "@flatfile/api";
+import { WorkflowType, getSpace, getWorkbook } from "../lib/flatfile";
+
+type WorkbookConfig = {
```

Comment:
Do we need to specify this type or can we use one from the package?

Reply:
It should be there, but I cant import it from Flatfile. Its there in the docs imported from Flatfile. Weird.

Reply:
![CleanShot 2023-06-14 at 14 00 53](https://github.com/FlatFilers/HCMShow/assets/12041898/3c8cbf40-4afb-48b5-b4c0-266a6ad36953)

Reply:
try `Flatfile.CreateWorkbookConfig` ?

Reply:
edited to new